index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>My Portfolio</title>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  </head>
+
+  <body>
+    <header class="header">
+        <a href="#" class="logo">My Logo</a>
+        <nav id="nav-hamburger">
+            <label class="ham-icon" for="hamburger">&#9776;</label>            
+        </nav>
+    </header>
+    <section id="main">
+      <div id="background">
+        <img src="images/Rectangle 6.png" class="shape1">
+        <img src="images/Ellipse 11.png" class="shape2">
+        <img src="images/Ellipse 5.png" class="shape3">
+        <img src="images/Rectangle 7.png" class="shape4">
+        <img src="images/Ellipse 8.png" class="shape5">
+        <img src="images/Rectangle 4.png" class="shape6">
+        <img src="images/Ellipse 3.png" class="shape7">
+        <img src="images/Ellipse 12.png" class="shape8">
+        <img src="images/Ellipse 13.png" class="shape9">
+        <img src="images/Ellipse 15.png" class="shape10">
+        <img src="images/Ellipse 4.png" class="shape11">
+        <img src="images/Rectangle 5.png" class="shape12">
+      </div>
+        <h1 class="main-heading">I'm Misal.<br> Glad to see you!</h1>
+        <p class="main-paragraph">I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+        <label class="connect">LET'S CONNECT</label>
+      <div id="socials">
+          <a href="#" class="icons"><img src="images/icons/twitter.png"></a>
+          <a href="#" class="icons"><img src="images/icons/Linkedin.png"></a>
+          <a href="#" class="icons"><img src="images/icons/M.png"></a>
+          <a href="#" class="icons"><img src="images/icons/github.png"></a>
+          <a href="#" class="icons"><img src="images/icons/5.png"></a>
+      </div>
```

Comment:
- [ ] Great job 👏 adding the social media icons, but as specified by the previous reviewer kindly place the links inside a `ul` tag as separate `li` elements as this ensures they are list items and helps supportive devices as screen readers identify it.